visRayTracer.mac

```diff
@@ -0,0 +1,33 @@
+# Visualization and setup macro 
+# This file is run by default if you just type WCSim and nothing else.
+
+/run/verbose 0
+/tracking/verbose 0
+/hits/verbose 0
+
+##select RayTracer Visualization
+/WCSim/WCVisChoice RayTracer
+
+## select the geometry
+# Default config if you do nothing is currently SuperK
+#/WCSim/WCgeom SuperK
+
+# Some other SuperK options:
```

Comment:
This comment on line 15 should be removed.

Also, there should be the /WCSim/Construct line here. Make sure you put a comment beside it that says that this line is needed for raytracer to work (people are used to uncommenting this only if they want to switch the configuration to something other than SK, but in this case it is needed for all detector configs).